cmd/create.js

```diff
@@ -88,18 +88,20 @@ function createOrg (orgname, argv) {
 
   results[numResults++] = shell.exec(createCommand)
 
-  if (results[numResults - 1].stderr || results[numResults - 1].stdout.indexOf('ERROR') != -1) {
+  let lastResult = results[numResults - 1];
+  if (lastResult.stderr || lastResult.stdout.indexOf('ERROR') != -1) {
     console.error(
       err('Could not create a new scratch org. Please ensure that the default developer hub is configured properly.')
     )
     process.exit(1)
   } else {
+    let username = /\S*@example.com/.exec(lastResult.stdout)[0];
```

Comment:
Is the username that can be used always @example.com?

Reply:
In my experience, it's always been "@example.com", but not sure if devhub admins can change it via settings. But good point, let me post another version that would work with other domains.

Reply:
I think that rather than looking for an email (which can get really complicated), we can just look at the config file if any is used and / or use a parameter for default username. I suppose if none of those is used, then maybe @example can be used?